benchmarks/scale_1k_20k/test_1k_20k.py

```diff
@@ -0,0 +1,93 @@
+import ray
+import ray.autoscaler.sdk
+
+import json
+import os
+from time import sleep, perf_counter
+from tqdm import tqdm, trange
+
+TEST_NUM_NODES = 1000
+MAX_ACTORS_IN_CLUSTER = 20000
+
+
+def num_alive_nodes():
+    n = 0
+    for node in ray.nodes():
+        if node["Alive"]:
+            n += 1
+    return n
+
+
+def scale_to(target):
```

Comment:
Are we supposed to test actor scheduling while we are autoscaling? Otherwise, can we split this logic into a separate script and add it to `prepare` field? That will help us isolating infra failures from the app level test failure

Reply:
.e.,g https://github.com/ray-project/ray/blob/7b1c207be33892df6e556e7b5f7fb4b44c81c821/release/nightly_tests/nightly_tests.yaml#L86